### DISCUSSION TITLE
Question: MRXS multichannel slide

### DISCUSSION THREAD

**Discussion by ilicmilos12896** [libvips/libvips#2952] [Q&A]
Hi! 
Is there a possibility to use libvips c++ API in order to process .mrsx images with more then 3 channels. I need to open and get regions from images with 4 and more channels. I also need to extract and color each channel separately.

Thanks in advance

**Comment by jcupitt** [comment]
Hi @ilicmilos12896,

I don't think openslide supports anything except RGB (it uses cairo internally for rendering), but I could be wrong. Do you have a sample slide with more than RGB?
